astropy/constants/codata2017.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Astronomical and physics constants in SI units.  See :mod:`astropy.constants`
+for a complete listing of constants defined in Astropy.
+"""
+import math
```

Comment:
p.s. CODATA 2014 used `numpy` for `np.pi` but I prefer to fall back to built-in `math` for such things. Any reason why `np.pi` is superior?

Reply:
As far as I know both are identical, so it really doesn't matter which one you use.